prototypes/entity.lua

```diff
@@ -59,16 +60,115 @@ data:extend({
 		{
 			shadow =
 			{
-				red = {2.01, 0.6},
-				green = {2.52, 0.6}
+				red = {2.52, 0.65},
+				green = {2.01, 0.65}
```

Comment:
What are these changes for? (I can't test them in-game right now.)

Reply:
All of the wiring changes were cosmetic and to make them look correct to please my OCD. Don't have to be implemented but I just wanted them to be closer to looking correctly attached.

Reply:
Like updating the item descriptions, these can be done in a separate pull request. That way the circuit-wire visual updates can be included in an upcoming maintenance release (the 0.2.x series) instead of waiting until this whole PR is ready to go in 0.3.0 or later.